aries-site/src/pages/templates/empty-state.mdx

```diff
@@ -0,0 +1,130 @@
+
+A well-designed empty state can improve the user experience and reduce confusion by setting clear expectations for what the user should do next.
+
+Here are a few characteristics that make a well-designed empty state:
+
+1.  **Clear messaging:** The empty state should have simple and clear messaging that tells the user what information is missing, why it is missing, and what they can do next to access the information. Follow the voice and tone guidance within the design system to remain clear and empathetic to the user.
+1.  **Visual cues:** The empty state should use iconography to help convey the message and create a more engaging user experience. The icon should be relevant to the context of the interface and help the user understand why it's empty.
+1.  **Guidance:** The empty state should provide guidance and a direct pathway out of the empty state when applicable. Avoid leading the user to a dead end.
+1.  **Calls-to-action:** The empty state should include calls-to-action that encourage the user to take action, such as creating their first resource or adding their first item.
+1.  **Help or documentation links:** The empty state may include links to help or documentation resources that can assist the user in populating the view with data or finding relevant information.
+
+## Anatomy
+
+<Example plain>
+  <EmptyStateAnatomy />
+</Example>
+
+Although there are many diverse use cases for empty states within our solutions, we use a single component structure to ensure they are consistently applied across the complete user experience. Below describes the structure of an empty state component:
+
+| Label | Region          | Purpose                                                             |                  Required                   | Notes                                                                      |
+| ----- | --------------- | ------------------------------------------------------------------- | :-----------------------------------------: | -------------------------------------------------------------------------- |
+| **1** | **Icon**        | Imagery to help the user process the message.                       |                                             | Use size="xlarge" on icon.                                                 |
+| **2** | **Heading**     | Communicates the system status.                                     | <Checkmark a11yTitle="true" size="small" /> | Use the semantic heading level that fits the placement of the empty state. |
+| **3** | **Description** | Guides the user out of the empty state                              | <Checkmark a11yTitle="true" size="small" /> | Use default text size of "medium".                                         |
+| **4** | **Action**      | A direct pathway out of the empty state.                            |                                             |                                                                            |
```

Comment:
For Action and Anchor, I would probably state within the notes to try and keep it to two or fewer.

Reply:
Done.

Reply:
Confirmed.